composer.json

```diff
@@ -19,7 +19,7 @@
         "Laravel", "Backpack", "Backpack for Laravel", "Backpack Addon", "ActivityLog"
     ],
     "require": {
-        "backpack/crud": "^5.3.13",
+        "backpack/crud": "^6.0|^5.3.13",
```

Comment:
Let's publish a v1.0.0 with support for both Backpack v5 and Backpack v6. It's zero-cost. 
Then right afterwards, we can create a v2.0.0 that drops support for BP v5.

Boom! We support both, but only push updates to v2.